manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -207,9 +207,19 @@ spec:
                     values:
                       - vsphere-csi-controller
               topologyKey: "kubernetes.io/hostname"
+        nodeAffinity:
```

Comment:
likewise can we skip csi node daemonset pods from running on control-plane nodes

Reply:
I am not sure if user will schedule application pods on control-plane nodes. @divyenpatel Can you please share your opinion on this.

Reply:
we can do that.


There will be impact in the way we are looking for shared datastores in the k8s cluster of in specific zone. 
If CSI Node Daemonsets pods are not running, then basically we will skip those nodes for finding shared datastore.

also we need to think about upgrade impact. What if CSInodes objects already created in the prior releases for control plane nodes and after upgrade we do not delete them.

Can we take this as separate PR?

Reply:
Yes, will not consider this change as part of this PR.